16/umbraco-cms/release-candidate-guide.md

```diff
@@ -2,46 +2,56 @@
 description: >-
   Learn how to start testing the Release Candidate for the latest version of
   Umbraco CMS, and find information about new and updated documentation.
-hidden: true
 ---
 
 # Release Candidate Guide
 
-## How to test the Release Candidate
+The can be used to test your website and projects against the next major version of Umbraco CMS.
```

Comment:
```suggestion
These can be used to test your website and projects against the next major version of Umbraco CMS.
```

Reply:
Missed adding "Release Candidate" after "The" 😅 
I'll commit the fix